tests/e2e.test.js

```diff
@@ -170,6 +159,23 @@ describe('Search', () => {
     },
   );
 
+  it('always show report buttons', async () => {
+    const $iconMissing = await page.$('#icon-missing');
+    const $iconOutdated = await page.$('#icon-outdated');
+    expect(await isVisible($iconMissing)).toBeTruthy();
+    expect(await isVisible($iconOutdated)).toBeTruthy();
+
+    const $searchInput = await page.$('#search-input');
+    await $searchInput.type(' ');
```

Comment:
This test is very confusing. Really is testing that typing a space in the search bar does not find anything, but this is not clear to me reading it. In fact, do not display icons when typing only whitespaces seems like a bug because if I add other letter after them, the icons reappear...